metpy/__init__.py

```diff
@@ -16,9 +16,5 @@
 __version__ = get_versions()['version']
 del get_versions
 
-try:
-    # Added in Python 3.2, will log anything warning or higher to stderr
-    logging.lastResort
-except AttributeError:
-    # Add our own for MetPy on Python 2.7
-    logging.getLogger(__name__).addHandler(logging.StreamHandler())
+# Added in Python 3.2, will log anything warning or higher to stderr
```

Comment:
Should be able to just nuke this entirely. Original code was just checking for this attribute and then doing more on 2.7 (where it didn't exist).